rcl/test/rcl/test_wait.cpp

```diff
@@ -27,6 +30,8 @@
 #endif
 
 
+#define TOLERANCE 100000  // clock error tolerance in nanoseconds
```

Comment:
Ah, just kidding there is... Past self is smarter than current self apparently. Yes those macros are the one's I was talking about.